### PR TITLE
add best practices and tips-and-tricks

### PR DESCRIPTION
**Pull request by ed-nykaza**

### CODE REVIEW

File: best-practices.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Best Practices
+
+## Backing up your computer/work
+ - Try to do any/all work in a github repository to take advantage of version control.
```
Comment (jameno) [inline best-practices.md:4]: Is the work being done in github just code? If so I might make that explicit here as

```suggestion
 - Try to do any/all coding work in a github repository to take advantage of version control.
```

**Review by jameno — CHANGES_REQUESTED** [review]
I also started moving some of our tips and tricks over and I ended up creating a dedicated tips-and-tricks folder. My branch is here: [jam/update-tips-and-tricks](https://github.com/tidepool-org/data-science-handbook/tree/jameno/update-tips-and-tricks/tips-and-tricks)

I think if you change `tips-and-tricks.md` to `about-tips-and-tricks.md` and move it into a **tips-and-tricks** folder we should avoid the future merge conflict.

**Review by aeevered — APPROVED** [review]
@ed-nykaza all looks good to me

File: best-practices.md (hunk 1 above)
Comment (ed-nykaza) [reply best-practices.md:4]: per our conversation, I am going to merge and let you push your changes on top of these.

**Pull request merged**